test/e2e/scalability/load.go

```diff
@@ -458,6 +458,8 @@ func GenerateConfigsForGroup(
 			MemRequest:     26214400, // 25MB
 			SecretNames:    secretNames,
 			ConfigMapNames: configMapNames,
+			// Define a label to group every 2 RCs into one service.
+			Labels: map[string]string{"svc-label": groupName + "-" + strconv.Itoa((i-1)/2)},
```

Comment:
define `svc-label` as a const? so that we don't need to hard-coded it everywhere.